django/db/models/lookups.py

```diff
@@ -55,32 +57,24 @@ def _unregister_lookup(cls, lookup):
         del cls.class_lookups[lookup.lookup_name]
 
 
-class Transform(RegisterLookupMixin):
+class Transform(RegisterLookupMixin, BaseExpression):
 
     bilateral = False
```

Comment:
Sorry for the half message, I'm on mobile.

The upside of constructor is that when we get the new syntax we can customize it per query! e.g. `F('name').unaccent(bilateral=False) == 'Loïc'`.